app/services/message-service.js

```diff
@@ -26,19 +23,11 @@ export const sendMessageToSingleFrontDoor = async (
     success
   )
 
-  return outboundMessage
-}
-
-const validateInboundMessage = (logger, inboundMessage) => {
-  const { error } = inboundMessageSchema.validate(inboundMessage, {
-    abortEarly: false
-  })
-
-  if (error) {
-    const errorMessage = `The inbound message is invalid. ${error.message}`
-    logger.error(errorMessage)
-    throw new Error(errorMessage)
+  if (!success) {
+    throw Error('See earlier error.')
```

Comment:
Not sure we should have errors that refer to other errors, just give the reason here?

Reply:
When comms with SFD fail, we delay throwing an error so we can store a message_log record first. Was hoping to refactor at a later date.

Reply:
Yes but for now put the reason for the error into the error rather than a suggestion to look elsewhere? A message like "sending message to SFD unsuccessful".

Reply:
Very true, thanks